code/modules/mob/_modifiers/traits_phobias.dm

```diff
@@ -416,13 +423,19 @@
 			fear_amount += 3
 
 		if(istype(thing, /obj/effect/alien/resin)) // Resin's a bit slimy according to its own description.
-			fear_amount += 1
+			if(severe)
+				fear_amount += 1
 
 		if(istype(thing, /obj/item/weed_extract))
+			if(severe)
+				fear_amount += 1
+
+		if(istype(thing, /obj/item/toy/plushie/slime)) //Squish squish.
 			fear_amount += 1
 
 		if(istype(thing, /obj/effect/decal/cleanable/mucus)) // Blennophobia apparently includes mucus, so!
-			fear_amount += 2
+			if(severe)
+				fear_amount += 2
 
 		if(istype(thing, /obj/item/slime_extract)) // Gooey.
 			fear_amount += 1
```

Comment:
Generalize this to a list associating types with fear amounts.
Make the severe versions append to the minor versions' lists of things (Arguably should overwrite b/c the fear amount from the actually-spooky-things should be higher)